src/jobs/lint-test-build.yml

```diff
@@ -31,8 +31,11 @@ executor:
   name: default
   tag: << parameters.version >>
 
+working_directory: <<parameters.working_directory>>
+
 steps:
-  - checkout
+  - checkout:
+      path: /home/circleci/project
```

Comment:
Is this working directory? Can we use the parameter value then of `<<parameters.working_directory>>`, Also if working directory is set above, I'm not sure this path needs to be set at all.

This might just be a language thing. Is this meant to be similar to the Node and Python Orb's similar `app-dir`
 - https://circleci.com/developer/orbs/orb/circleci/node#commands-install-packages
 - https://circleci.com/developer/orbs/orb/circleci/python#commands-install-packages
 
 If so, I'd like to keep this in line with the other language orbs and use the same naming conventions.

Reply:
@KyleTryon Since this is an opinionated job, we enforce the checkout location. When we set the working directory, it clones it to that location, defeating the purpose of the fix for #6 - which is to say, caching :)